core/timetrace.go

```diff
@@ -56,7 +56,7 @@ func New(config *config.Config, fs Filesystem) *Timetrace {
 // Since parallel work isn't supported, the previous work must be stopped first.
 func (t *Timetrace) Start(projectKey string, isBillable bool) error {
 	latestRecord, err := t.LoadLatestRecord()
-	if err != nil {
+	if err != nil && !errors.Is(err, ErrAllDirectoriesEmpty) {
```

Comment:
I tested with this and it still fails, but works with 
`if err != nil && !errors.Is(err, ErrTrackingNotStarted) {`

Reply:
Pretty strange since `LoadLatestRecord` should only return an `ErrAllDirectoriesEmpty` (from latestNonEmptyDir`) 🤔

Reply:
Let me try again, I might have missed something.

Reply:
When I check of `!errors.Is(err, ErrTrackingNotStarted)`, I get an error:

```
❗ Failed to start tracking: all directories empty
```

This indicates that `ErrAllDirectoriesEmpty` is returned.

Reply:
checking for the `ErrAllDirectoriesEmpty` works fine for me too if `LoadLatestRecord` returns the error straight from `latestNonEmptyDir`

Reply:
@KonstantinGasser @dominikbraun  Right, this was my mistake, I hadn't updated my record.go file, apologies.